### DISCUSSION TITLE
disable config validator temporary

### DISCUSSION THREAD

**Discussion by erth5** [ash-jc-allen/laravel-config-validator#56] [General]
I get errors by start run Laravel Dusk in a local environment
Is there a possibility to deactivate config validator temporary without remove from composer.json?
Or Did I have configure dusk ?

The config is correct but changed while dusk runs
`
$php artisan dusk
Environment modified. Restarting server...
....
Environment modified. Restarting server...
`

![failure-Tests_Browser_DebugTest_testRidirect-0](https://user-images.githubusercontent.com/9325448/179996636-aeb30025-ffc5-4126-90c3-726a1960bf1a.png)

**Comment by ash-jc-allen** [comment]
Hey! Sorry that you're having issues with this. Where are you calling the validator from? I see that you have the `AppServiceProvider@boot` method in your stack trace. Are you calling it from there? 🙂

**Comment by ash-jc-allen** [comment]
I'm going to close this issue for the time being so that I can keep on top of my open issues. But, if you're still having this issue, please feel free to reopen the issue. Thanks! 😄

**Comment by erth5** [comment]
I found the Cause, I used

        if (App::environment() === 'local') {

            (new ConfigValidator())
                ->run();
        }
        
to check the config on startup, but Dusk use own environment

**Comment by ash-jc-allen** [comment]
Perfect! I'm glad that you managed to get this sorted 😄
